src/utils.ts

```diff
@@ -32,6 +32,7 @@ export function hasProperty(object: object, key: string) {
 }
 
 export function isJestEnv(): boolean {
+  // @ts-ignore Do not use `@types/node` because it will break setTimeout function types in React Native projects
```

Comment:
Hey @henrikra, 
Thanks for doing this! 🙌 
As it's not only breaking the `setTimeout` but also other stuff like `setInterval`, I would rewrite this comment in this  way:
```
Using `@types/node` will prioritise node types over RN types which can break the types in React Native projects.
```